versions.go

```diff
@@ -3,25 +3,24 @@ package resource
 import (
 	"regexp"
 	"strconv"
+	"strings"
 
 	"github.com/google/go-github/github"
 )
 
 // determineVersionFromTag converts git tags v1.2.3 into semver 1.2.3 values
 func determineVersionFromTag(tag string) string {
-	re := regexp.MustCompile("v?([^v].*)")
+	re := regexp.MustCompile(`^v?(\d+\.)?(\d+\.)?(\*|\d+.*)$`)
```

Comment:
Since this is not using `TagNameRegex` are you guaranteed to have values return correctly?